tic_tac_toe/game.py

```diff
@@ -87,18 +129,61 @@ def move(game, player, position):
     checks before the actual movement is done.
     After registering the movement it must check if the game is over.
     """
-    pass
+    board = game['board']
+    if game['winner'] or _board_is_full(board):
+        raise InvalidMovement("Game is over.")
+        
+    elif not _position_is_valid(position):
+        raise InvalidMovement('Position out of range.')
+    
+    elif not _position_is_empty_in_board(position, board):
+        raise InvalidMovement('Position already taken.')
+    
+    elif get_next_turn(game) != player:
+        raise InvalidMovement('"{}" moves next'.format(game['next_turn']))
+    
+    
+    board[position[0]][position[1]] = player
+    
+    winner = _check_winning_combinations(board, player)
+    
+    if winner:
+        game['winner'] = winner
```

Comment:
Instead of setting `winner` and then like two lines later assigning it to `game['winner']` you could just set it directly: `game['winner'] = _check_winning_combinations(board, player)` This works because `_check_winning_combinations` returns None if there is no winner.